docker-compose.yml

```diff
@@ -25,7 +25,7 @@ services:
       - PYTHON_CODEGEN=off
       - REDIS_QUEUE_HOST=redis-queue
       - REDIS_STATE_HOST=redis-state
-      - LD_LIBRARY_PATH=/build/faasm/third-party/lib
+      - LD_LIBRARY_PATH=/build/faasm/third-party/lib:/usr/local/lib
```

Comment:
This seems to be required to cover both local development and a fresh docker-compose set-up